app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -1198,12 +1209,13 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
         }
 
     @SuppressLint("CheckResult")
-    private void onDeleteClicked(Spinner spinner) {
+    public void onDeleteClicked(Spinner spinner) {
```

Comment:
Any specific reason why this needs to be public?

Reply:
actually wanted to do it locally but instead pushed it by mistake I will make it private